Products/Archetypes/skins/archetypes/widgets/js/textcount.js

```diff
@@ -1,12 +1,13 @@
 <!-- Original:  Ronnie T. Moore -->
 <!-- Dynamic 'fix' by: Nannette Thacker -->
 function textCounter(field, countfield, maxlimit) {
-  var fieldval = jQuery(field).attr('value');
+  var fieldval = jQuery(field).val();
+  var countfield = jQuery('input[name="' + countfield + '"]');
   if (fieldval.length > maxlimit) {
       // if too long...trim it!
-      jQuery(field).attr('value',  fieldval.substring(0, maxlimit));
-      alert( 'This field is limited to ' + maxlimit + ' characters in length.' );
-  } 
-  // update 'characters left' counter	
-  jQuery('input[name="' + countfield + '"]').attr('value', Math.max(maxlimit - fieldval.length, 0));
+      jQuery(field).val(fieldval.substring(0, maxlimit));
+      countfield.css('border', '1px solid red');
+  }
+  // update 'characters left' counter
+  countfield.attr('value', Math.max(maxlimit - fieldval.length, 0));
```

Comment:
A few lines up you replace `jQuery(...).attr('value', '...')` with `jQuery(...).val('...')`. Shouldn't you do the same here?
Note: I have no idea if there are cases where one works and the other doesn't.